### DISCUSSION TITLE
convert 2d floorplan to 3d

### DISCUSSION THREAD

**Discussion by WWGate** [zcemycl/TF2DeepFloorplan#28] [Q&A]
Hello and thank you for your amazing effort and paper.

Could you please let me know how to extract the coordinate of each objects classes in order to render them as 3d scenes?

Any help is highly appreciate it

Best 
Fadi

**Comment by zcemycl** [comment]
@WWGate Unfortunately, I don't see any source code related to this from their original repo . What you could try is using the colormap output from `--postprocess` to locate the pixels with certain room type. You can also use the windows and doors as the landmarks for identifying projection from 2d to 3d.

**Comment by WWGate** [comment]
@zcemycl thank you for your kind reply.
I'll do my best to figure it out

**Comment by zcemycl** [comment]
@WWGate You can try the new pygame raycast :)
